.github/workflows/sign-off.yml

```diff
@@ -0,0 +1,76 @@
+#
+#  Copyright (C) 2024 The Draupnir Project
+#  All rights reserved.
+#
+#  This file is modified and is NOT licensed under the Apache License.
+#  This modified file incorperates work from matrix-org/backend-meta
+#  https://github.com/matrix-org/backend-meta
+#  which included the following license notice:
+#
+#Copyright 2022 - 2024 The Matrix.org Foundation C.I.C.
+#
+#Licensed under the Apache License, Version 2.0 (the "License");
+#you may not use this file except in compliance with the License.
+#You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#Unless required by applicable law or agreed to in writing, software
+#distributed under the License is distributed on an "AS IS" BASIS,
+#WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#See the License for the specific language governing permissions and
+#limitations under the License.
+#
+#  However, this file is modified and the modifications in this file
+#  are NOT distributed, contributed, committed, or licensed under the Apache License.
+#
```

Comment:
```suggestion
// Copyright 2024 Catalan Lover <catalanlover@protonmail.me>
// Copyright 2022 - 2024 The Matrix.org Foundation C.I.C.
//
// SPDX-License-Identifier: Apache-2.0
//
// SPDX-FileAttributionText: <text>
// This modified file incorporates work from matrix-org/backend-meta
// https://github.com/matrix-org/backend-meta
// </text>
```

Reply:
^ just do this it's simpler, keep it under apache there's no reason to incorporate this.